src/main/java/ttt/CommandLineGameController.java

```diff
@@ -26,6 +31,7 @@ public CommandLineGameController(GameRules gameRules, Prompt commandLinePrompt)
     public static void main(String... args) {
         CommandPrompt gamePrompt = buildPrompt();
         CommandLineGameController commandLineGameController = new CommandLineGameController(
+                new TicTacToeGameConfiguration(),
                 buildGameRules(gamePrompt),
```

Comment:
Nice thing about the tension I was talking about earlier is that "building the game rules" makes more sense (_semantically speaking_) inside the `GameConfiguration`.